sql/parser/sql.y

```diff
@@ -3211,6 +3220,8 @@ extract_arg:
 | HOUR {}
 | MINUTE {}
 | SECOND {}
+| MILLISECOND {}
```

Comment:
You can remove the ` {}` suffix from all these lines. The default action is what we want. That this works with an empty action is an accident.